packages/wrangler/src/publish.ts

```diff
@@ -61,7 +60,10 @@ export default async function publish(props: Props): Promise<void> {
   }
 
   let scriptName = props.script ? props.name : config.name;
-  scriptName += props.env ? `-${props.env}` : "";
+  if (props.legacyEnv) {
```

Comment:
I don't think we should be defaulting to Services until we figure out a transition plan.

Reply:
It's not the default in the current PR; it has to be explicitly enabled. I can leave it for a future PR though.